common/shared.c

```diff
@@ -496,7 +491,6 @@ int ft_finalize(
 	msg.desc = NULL;
 	msg.iov_count = 1;
 	msg.addr = addr;
-	msg.context = &tx_ctx;
```

Comment:
Why was the tx_ctx removed?  This won't work with providers that require using fi_context.